avocado/core/sysinfo.py

```diff
@@ -213,6 +214,45 @@ def stop(self):
         return retcode
 
 
+class JournalctlWatcher(Collectible):
+
+    """
+    Track the content of systemd journal into a compressed file.
+
+    :param logf: Basename of the file where output is logged (optional).
+    """
+
+    def __init__(self, logf=None):
+        if not logf:
+            logf = 'journalctl.gz'
+
```

Comment:
Yep, but since this style fix would touch other code also, we can do it after this.